sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -737,10 +793,10 @@ def _read(self, n, initial=False, buffer=None, _errnos=None):  # pylint: disable
                         self._read_buffer = BytesIO(data[n:])
                         n = 0
                 return view
-            except AttributeError:
-                raise IOError("Websocket connection has already been closed.")
+            except AttributeError as exc:
```

Comment:
from None?

Reply:
yes from None. websocket raises an attribute error when trying to get bytes off the wire, but ends up getting nothing.